src/components/BrowseSkill/BrowseSkill.js

```diff
@@ -261,23 +263,23 @@ export default class BrowseSkill extends React.Component {
                         description = 'No description available'
                     }
                     if (skill.skill_rating) {
-                        average_rating = parseInt(skill.skill_rating.stars.avg_star,10);
+                        average_rating = parseFloat(skill.skill_rating.stars.avg_star,10);
```

Comment:
Is the `,10` param still relevant? :thinking:

Reply:
Have fixed it out. Thanks for pointing it out :+1: 
It gives a warning is parseInt without radix as the number can be Hex, Decimal, etc.
But praseFloat can only mean Decimal, hence the radix not required.